Sources/UI/UITableView+Diff.swift

```diff
@@ -1,5 +1,5 @@
-#if canImport(OptiDiff)
-  import OptiDiff
+#if SWIFT_PACKAGE
```

Comment:
Nice! 
Please run `./swiftformat .` for your changes.

Reply:
Thanks. Should be fine now.